my-ansible-playbook.yml

```diff
@@ -0,0 +1,73 @@
+- name: Install To Do App on new web server
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: trello_api_key
+      prompt: Trello API Key
+
+    - name: trello_api_token
+      prompt: Trello API Token
+
+    - name: trello_board_id
+      prompt: Trello Board ID
+
+  tasks:
+  - name: Install Git
+    ansible.builtin.yum:
+      name: git
+      state: present
+    become: yes
+
+  - name: Install Python3
+    ansible.builtin.yum:
+      name: python3
+      state: present
+
+  - name: Install Poetry
+    ansible.builtin.shell:
+      cmd: curl -sSL https://install.python-poetry.org | python3 -
+      creates: /usr/local/bin
```

Comment:
The purpose of `creates` on the shell task is to say "this task should be skipped if <file> already exists".

So it should point to the poetry file. If it's something general like /usr/local/bin, then it's always true that it exists so this task will always get skipped.

Reply:
Changed to point to the poetry file.